indico/modules/rb_new/client/js/actions.js

```diff
@@ -88,6 +90,16 @@ export const OPEN_FILTER_DROPDOWN = 'OPEN_FILTER_DROPDOWN';
 export const CLOSE_FILTER_DROPDOWN = 'CLOSE_FILTER_DROPDOWN';
 
 export const SET_BOOKING_AVAILABILITY = 'SET_BOOKING_AVAILABILITY';
+
+// Blockings
+export const FETCH_BLOCKINGS_STARTED = 'FETCH_BLOCKINGS_STARTED';
+export const FETCH_BLOCKINGS_FAILED = 'FETCH_BLOCKINGS_FAILED';
+export const UPDATE_BLOCKINGS = 'UPDATE_BLOCKINGS';
+export const CREATE_BLOCKING = 'CREATE_BLOCKING';
+export const BLOCKING_CONFIRMED = 'BLOCKING_CONFIRMED';
```

Comment:
what about using `<whatever>_(REQUEST|SUCCESS|ERROR)` for the request-related actions? especially here for blockings, CONFIRMED is very misleading. i just forgot to rename it for the booking actions (where it's also misleading, "confirmed" sounds like accepting a prebooking) when adding the ajax util and added the three names mentioned above for `GET_BOOKING_AVAILABILITY_*`